WcaOnRails/spec/models/user_spec.rb

```diff
@@ -87,11 +78,6 @@
     expect(delegate).to be_valid
   end
 
-  it "requires senior delegate if delegate status allows it" do
-    delegate = FactoryBot.build :delegate, region_id: nil
-    expect(delegate).to be_invalid_with_errors(region_id: ["can't be blank"])
-  end
```

Comment:
The test is not needed because senior_delegate is replaced with region and the test for that is the previous test.